### DISCUSSION TITLE
Blank page hosted on Github Pages

### DISCUSSION THREAD

**Discussion by GuillaumeChk** [quasarframework/quasar#15814] [General - Components / Directives / etc] [answered]
The app is very simple, SPA with Quasar CLI with Vite.

Everything is fine when previewing production build (quasar build then quasar serve in dest/spa).

When I try to host it on GitHub Pages (dist/spa) I get a blank screen with error 404 for the files  …/assets/indexXXXXX.js and the css one too.

Public repo is here [https://github.com/GuillaumeChk/gslr-quasar-cli](https://github.com/GuillaumeChk/gslr-quasar-cli) and hosted pages here [https://guillaumechk.github.io/gslr-quasar-cli/](https://guillaumechk.github.io/gslr-quasar-cli/)

I tried to remove the favicon and icons, still same error it is not about images. I don't know what is wrong as there is no explicit error and I believe I have done everything right.

**Comment by dongwa** [comment, marked as answer]
Trying to edit your quasar.config.js:
```js
extendViteConf (viteConf) {
	viteConf.base = './'
},
```

**Reply by GuillaumeChk** [reply]
That was as simple. 
Thank you dongwa!

How did you found that? Did I miss something?

**Reply by dongwa** [reply]
> That was as simple. Thank you dongwa!
> 
> How did you found that? Did I miss something?

It's not in Quasar's documentation. You need to be familiar with Vite. Refer to https://vitejs.dev/config/shared-options.html#base

**Reply by JakubStasiak00** [reply]
Just had the same issue with my repo, thanks for your wisdom.
